addons/html_editor/static/src/others/embedded_components/plugins/file_plugin/file_plugin.js

```diff
@@ -1,103 +0,0 @@
-import { Plugin } from "@html_editor/plugin";
```

Comment:
Stable retrocompatibility: Same. Put back this file with a comment explaining it's not used anymore but is kept due to the stable policy.